Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -810,13 +809,13 @@ void CClientVehicle::Fix()
 
     SFixedArray<unsigned char, MAX_DOORS> ucDoorStates;
     GetInitialDoorStates(ucDoorStates);
-    for (int i = 0; i < MAX_DOORS; i++)
+    for (unsigned char i = 0; i < MAX_DOORS; i++)
```

Comment:
uint8_t has no sence. Two reasons: compiler will change this to uint32_t(size of a register) and SFixedArray::operator[] takes uint32_t. It's better to use size_t here.

Reply:
The same below and above